ossindex/ossindex.go

```diff
@@ -50,6 +50,11 @@ func getDatabaseDirectory() (dbDir string) {
 	return usr.HomeDir + "/.ossindex"
 }
 
+// RemoveCacheDirectory deletes the local database directory.
+func RemoveCacheDirectory() error {
+	return os.RemoveAll(getDatabaseDirectory())
```

Comment:
I'm wondering if there is a better way to do this (basically just flush Badger rather than delete the directory)

Reply:
Wonderbar.

I've been wanting to move away from Badger for a lil bit (it's a bit overkill for what we need, to be honest). You've got a reasonable explanation for now! Thanks for answering!